src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -124,7 +124,7 @@
   (if (= type :datemark)
     ^{:key (str "datemark" message-id)}
     [message.datemark/chat-datemark value]
-    (when (= content-type constants/text-content-type)
+    (when (some #{content-type} [constants/text-content-type constants/content-type-emoji])
```

Comment:
Let's extract the vector of valid content types in a constant, too. Not only it would be useful, but it would make this code more readable - for a few moments I thought that `constants/text-content-type` is being called like a reagent component.